rdkit/Chem/Fingerprints/FingerprintMols.py

```diff
@@ -92,7 +88,7 @@ def FingerprintsFromSmiles(dataSource, idCol, smiCol, fingerprinter=Chem.RDKFing
       res.append((ID, fp))
       nDone += 1
       if reportFreq > 0 and not nDone % reportFreq:
-        message('Done %d molecules\n' % (nDone))
+        message('Done %d molecules\n' % nDone)
```

Comment:
Convert to f string. There are quite a few more places in the code that use %

Reply:
OK, I got your idea! I would fix it in the next patch. Thank you for your review.